_docs_use-cases/use-case-consumption-delete-a-peer-shared-attribute-and-notify-peer.md

```diff
@@ -0,0 +1,60 @@
+---
+# Start automatic generation
+permalink: use-case-consumption-delete-a-peer-shared-attribute-and-notify-peer
+published: true
+title: "Delete a peer shared Attribute and notify peer"
+type: use-case
+toc: true
+sidebar:
+  - title: "Integrate Enmeshed"
+    nav: "docs_integrate"
+properties:
+  - id: RA24
+  - component: Runtime
+  - layer: Consumption
+  - facade: AttributesFacade
+  - function: deletePeerSharedAttributeAndNotifyPeer
+  - description:
+  - feature category: Cross-identity attribute sharing
+  - tech category: Attributes
+  - status: DONE
+  - documentation status: DONE
+  - comments:
+  - actor: Identity
+  - trigger: REST API
+  - precondition:
+  - result:
+  - priority: n/a
+  - complexity: n/a
+  - size: n/a
+  - created_at:
+  - changed_at:
+  - api_route_regex:
+  - published: default
+  - link: use-case-consumption-delete-a-peer-shared-attribute-and-notify-peer
+require:
+required_by:
+# End automatic generation
+---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case allows you to delete a [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) a peer shared with you.
+
+## Parameters
+
+- The `attributeId` of the peer shared Attribute you want to delete.
+
+## On Success
+
+- The peer shared Attribute will be deleted.
+- All predecessors of the peer shared Attribute will be deleted.
+- If the peer shared Attribute was succeeded, the `succeeds` property of the successor will be set to undefined.
+- A [Notification]({%link _docs_integrate/data-model-overview.md %}#notification) is sent to the owner of the Attribute, informing them that you deleted their Attribute. Technically, the own shared Attribute of the owner and all predecessors will get a `deletionInfo` with `deletionStatus` `"DeletedByPeer"` and the time of receiving the Notification as `deletionDate`.
```

Comment:
"A [Notification]({%link _docs_integrate/data-model-overview.md %}#notification) is sent to the owner of the Attribute." - Can this always be the case? What is about the ThirdPartyRelationshipAttributes, when the third party is the owner of it? We do not necessarily have a corresponding own shared Attribute counterpart for every peer shared Attribute. This could also lead to problems in other parts of the documentation and implementation.

Reply:
The deletion of third party owned RelationshipAttributes now has its dedicated use case. So, calling `DeletePeerSharedAttributeAndNotifyPeer` will in fact always send a Notification to the owner, which in case of peer shared Attributes by definition also is the peer. This will also be done, if the owner already deleted their corresponding own shared Attribute. Of course, then no `deletionInfo` will be set. However, I don't think it is necessary to give this level of detail here, as it would probably distract the reader from the important information of this point.

Reply:
All right. 👍 So will documentation for the third party owned RelationshipAttribute and notify peer use case and a corresponding section for the "Delete an Attribute" scenario be added to this PR?

Reply:
Yess, I'm on it.